packages/eventcatalog/package.json

```diff
@@ -19,6 +19,10 @@
     "@headlessui/react": "^1.4.2",
     "@heroicons/react": "^1.0.5",
     "@mdx-js/react": "^1.6.22",
+    "@stoplight/json-schema-viewer": "^4.5.0",
```

Comment:
@thim81 quick question, just looking at their docs, how did you know we need all these dependencies? 

Looking at docs looks like it's just the one, but looks like it doesnt work with just the one either!

Reply:
When trying to make it work, I had only the `@stoplight/json-schema-viewer` but then the React was complaining about missing packages, so I added them 1 by 1.